### DISCUSSION TITLE
Add predefined curves

### DISCUSSION THREAD

**Discussion by lukasoppermann** [primer/prism#16] [Ideas]
Colorbox.io has a nice feature where you can use computed curves: https://colorbox.io

A combination of choosing a computed curve and tweaking it afterwards would be 🤯

<img width="180" alt="Screenshot 2022-06-15 at 10 25 28" src="https://user-images.githubusercontent.com/813754/173780168-dcf58fea-aa50-4cea-bc04-4e803f1ba18a.png">

So basically the bottom two dropdowns for each section allow users to select a curve model like linear, cubic, quad, ... the other one allows you to choose between easeInAndOut, easeIn and easeOut.

I think this would help to find a better starting point. Maybe to begin with it could be combined in one option e.g. cubicEaseIn, cubicEaseOut and added to the dropdown that stores the user defined curves.

**Comment by colebemis** [comment]
Love this idea. Thanks for the suggestion, @lukasoppermann!

**Comment by colebemis** [comment]
@stevenpetryk added the ability to apply predefined easing functions to shared curves in [this PR](https://github.com/primer/prism/pull/43). Is this what you had in mind, @lukasoppermann?

**Reply by lukasoppermann** [reply]
Hey, I think this is exactly I was looking for. 👍
